.github/workflows/main-build.yml

```diff
@@ -151,7 +151,11 @@ jobs:
         config_mod: .version := "${{ inputs.version || format('{0}_build', github.ref_name) }}"
         parallel: true
         query: ^workflows
-  
+
+    - name: "Remove test component and workflows from the target folder"
+      run: |
+        rm -r target/_test
```

Comment:
Alternatively, we could add a config mod that sets .status to disabled if the scope is test.

Or we could add a flat to `viash ns build` -- should create a github issue for that

Reply:
> Alternatively, we could add a config mod that sets .status to disabled if the scope is test.

I tried, but it does not work. For some reason filtering on a sublevel and assigning a new value based on that is not supported by the config mods. E.g. this does not work:
`--config_mod '.[scope.target == "test"].status := "disabled"'`

Also something like this does not work:

`'.[.name == "download_file"].namespace := "foo"'`

> Or we could add a flat to `viash ns build` -- should create a github issue for that
I also made that suggestion to @Grifs , but we descided to see if there is a broader usecase for this,